x-pack/plugins/apm/server/lib/helpers/get_document_sources.ts

```diff
@@ -5,11 +5,50 @@
  * 2.0.
  */
 import { kqlQuery, rangeQuery } from '@kbn/observability-plugin/server';
-import { ApmDataSource } from '../../../common/data_source';
+import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
 import { ApmDocumentType } from '../../../common/document_type';
 import { RollupInterval } from '../../../common/rollup';
 import { APMEventClient } from './create_es_client/create_apm_event_client';
 import { getConfigForDocumentType } from './create_es_client/document_type';
+import { TRANSACTION_DURATION_SUMMARY } from '../../../common/es_fields/apm';
+import { TimeRangeMetadata } from '../../../common/time_range_metadata';
+
+const getFilters = ({
```

Comment:
this doesn't return just filters, but it returns the request. Can you rename it?